source/js/classes/crop-host.js

```diff
@@ -720,6 +720,7 @@ crop.factory('cropHost', ['$document', '$q', 'cropAreaCircle', 'cropAreaSquare',
         };
 
         this.setAspect = function(aspect) {
+            isAspectRatio=true;
```

Comment:
@CrackerakiUA i think it is much proper place to set `isAspectRatio` , isn't it ?